bindings/go/src/_stacktester/stacktester.go

```diff
@@ -865,6 +865,16 @@ func (sm *StackMachine) processInst(idx int, inst tuple.Tuple) {
 
 	case strings.HasPrefix(op, "DIRECTORY_"):
 		sm.de.processOp(sm, op[10:], isDB, idx, t, rt)
+	case strings.HasPrefix(op, "GET_ESTIMATED_RANGE_SIZE"):
```

Comment:
For this one, you can check equality rather than whether the prefix matches. I'd also recommend moving it up, as the cases listed at the bottom are intended to be some special cases. Maybe up above `UNIT_TESTS` is sufficient? Ideally I'd say to match it with the list of operations in the spec (bindingApiTester.md), but it seems that the order of operations here is already quite different from that.

Reply:
Done